dali/python/nvidia/dali/ops.py

```diff
@@ -33,8 +33,9 @@
         ScalarConstant as _ScalarConstant, \
         Constant as _Constant
 
-
 cupy = None
+
+
```

Comment:
can we somehow make it more visible that this global variable is tightly coupled to the function below? would `noqa` in this line help?

Reply:
OK, fair enough.